File: Majority Element/majority element counter.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        majority_element_counter = 0
        for i in range(0, len(nums)):
            for j in range(i+1, len(nums)):
                if nums[i] == nums[j]:
                    majority_element_counter += 1
                    output = nums[i]
        return output

File: Majority Element/majority element counter.cc
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int output;
        int majority_element_counter = 0;
        for (int i=0; i<nums.size(); i++) {
        for (int j = i+1; j<nums.size(); j++) {
            if (nums[i] == nums[j]) {
                majority_element_counter +=1;
                output = nums[i];
            }
        }
        }
        return output;
    }
};